### PR TITLE
Create issue templates

### PR DESCRIPTION
**Pull request by Tlacenka**
In this PR, I create issue templates for the following:
- Feature request within a plugin.
- Change request for a plugin.
- New configuration request.
- Bug report.

🙏 Before merging, the issues feature should be checked in Settings. Otherwise, the YML format should hopefully work out of the box (I included `config.yml`).
image

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    validations:
+      required: true
+  - type: textarea
+    id: plugin-versions
+    attributes:
+      label: Plugin and dependencies version
+      description: What plugin version are you running? Include peer dependencies if applicable.
+      placeholder: |
+        For example:
+        `@code-pushup/eslint-config/typescript` version 0.5.0
```
Comment (matejchalk) [inline .github/ISSUE_TEMPLATE/bug-report.yml:62]: There's only one NPM package:

```suggestion
        `@code-pushup/eslint-config` version 0.5.0
```

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        - Vitest
+        - Cypress
+        - Storybook
+    validations:
+      required: true
+  - type: textarea
+    id: plugin-versions
+    attributes:
+      label: Plugin and dependencies version
+      description: What plugin version are you running? Include peer dependencies if applicable.
```
Comment (matejchalk) [inline .github/ISSUE_TEMPLATE/bug-report.yml:59]:
```suggestion
      label: Config and peer dependencies versions
      description: What config version are you running? Include peer dependencies if applicable.
```

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+      placeholder: |
+        For example:
+        `@code-pushup/eslint-config/typescript` version 0.5.0
+
+        Peer dependencies:
+        - @typescript-eslint/eslint-plugin version 7.4.0
```
Comment (matejchalk) [inline .github/ISSUE_TEMPLATE/bug-report.yml:65]:
```suggestion
        - `eslint` version 8.57.0
        - `@typescript-eslint/eslint-plugin` version 7.3.1
        - `@typescript-eslint/parser` version 7.3.1

```

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+      label: ESLint version
+      description: What ESLint version are you running? (check in your `package.json`)
+      placeholder: e.g. 8.57.0
+    validations:
+      required: false
+  - type: input
+    id: ts-version
+    attributes:
+      label: TypeScript version
+      description: What TypeScript version are you running? (check with `tsc -v` or in `tsconfig.base.json`)
```
Comment (matejchalk) [inline .github/ISSUE_TEMPLATE/bug-report.yml:80]: The version isn't mentioned in `tsconfig.json`.

```suggestion
      description: What TypeScript version are you running? (check with `npx tsc -v`)
```

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+        Peer dependencies:
+        - @typescript-eslint/eslint-plugin version 7.4.0
+    validations:
+      required: true
+  - type: input
+    id: eslint-version
+    attributes:
+      label: ESLint version
+      description: What ESLint version are you running? (check in your `package.json`)
```
Comment (matejchalk) [inline .github/ISSUE_TEMPLATE/bug-report.yml:72]:
```suggestion
      description: What ESLint version are you running? (check with `npm list eslint`)
```

File: .github/ISSUE_TEMPLATE/new-config-request.yml [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+    validations:
+      required: true
+  - type: textarea
+    id: acceptance-criteria
+    attributes:
+      label: Acceptance criteria
+      description: What should the new configuration contain? Listing a set of criteria helps us determine whether the scope of the request was fully addressed.
+      placeholder: |
+        For example:
+        - [ ] A new configuration `@code-pushup/eslint-config/react` includes the following rules: `eslint-plugin-react/recommended`, …
```
Comment (matejchalk) [inline .github/ISSUE_TEMPLATE/new-config-request.yml:20]:
```suggestion
        - [ ] A new configuration `@code-pushup/eslint-config/react` extends `plugin:react/recommended` with additional rules …
```

**Review by matejchalk — COMMENTED** [review]
Thanks a lot :+1:

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 3 above)
Comment (Tlacenka) [reply .github/ISSUE_TEMPLATE/bug-report.yml:65]: Should I include `eslint` as a peer dependency here or leave it separately as an input (see below lines 69+)

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 4 above)
Comment (Tlacenka) [reply .github/ISSUE_TEMPLATE/bug-report.yml:80]: You are right, it only references ES version.

**Review by matejchalk — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 3 above)
Comment (matejchalk) [reply .github/ISSUE_TEMPLATE/bug-report.yml]: OK, technically it's also a peer dependency, but having a separate input for it is fine also :shrug:

**Pull request merged**